django/utils/html.py

```diff
@@ -59,15 +59,18 @@ def escapejs(value):
 }
 
 
-def json_script(value, element_id=None):
+def json_script(value, element_id=None, json_encoder=None):
     """
     Escape all the HTML/XML special characters with their unicode escapes, so
     value is safe to be output anywhere except for inside a tag attribute. Wrap
     the escaped JSON in a script tag.
     """
-    from django.core.serializers.json import DjangoJSONEncoder
+    if json_encoder is None:
+        from django.core.serializers.json import DjangoJSONEncoder
 
-    json_str = json.dumps(value, cls=DjangoJSONEncoder).translate(_json_script_escapes)
+        json_encoder = DjangoJSONEncoder
+
+    json_str = json.dumps(value, cls=json_encoder).translate(_json_script_escapes)
```

Comment:
I think we can simplify this:
```python
def json_script(value, element_id=None, json_encoder=None):
    from django.core.serializers.json import DjangoJSONEncoder

    json_str = json.dumps(value, cls=json_encoder or DjangoJSONEncoder).translate(_json_script_escapes)
```

Reply:
That makes sense. I'll update that check.